esp-hal/src/usb_serial_jtag.rs

```diff
@@ -314,7 +314,7 @@ where
     fn new_inner(usb_device: impl Peripheral<P = USB_DEVICE> + 'd) -> Self {
         // Do NOT reset the peripheral. Doing so will result in a broken USB JTAG
         // connection.
-        PeripheralClockControl::enable(crate::system::Peripheral::UsbDevice);
+        PeripheralClockControl::enable(crate::system::Peripheral::UsbDevice, true);
```

Comment:
Just noting that we'll have to make sure dropping UsbSerialJtag doesn't reset the peripheral. Which is borderline impossible to do now :)